controllers/provider-aws/example/controlplane.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: extensions.gardener.cloud/v1alpha1
+kind: Cluster
+metadata:
+  name: default
+spec:
+  cloudProfile:
+    apiVersion: garden.sapcloud.io/v1beta1
+    kind: CloudProfile
+  seed:
+    apiVersion: garden.sapcloud.io/v1beta1
+    kind: Seed
+  shoot:
+    apiVersion: garden.sapcloud.io/v1beta1
+    kind: Shoot
+    spec:
+      cloud:
+        profile: aws
+        aws:
+          networks:
+            pods: 10.250.0.0/19
+      kubernetes:
+        version: 1.13.4
+      hibernation:
+        enabled: false
+    status:
```

Comment:
status should not be put in the resource.

Reply:
This is the status of the `Shoot` embedded in `Cluster`, not the status of the `Cluster` itself. It's also exactly the same as in https://github.com/gardener/gardener-extensions/blob/3e20d823b9c50a974f835c7e94a7669d5800615c/controllers/provider-aws/example/Infrastructure.yaml#L16. It also could be important, e.g. if the Shoot is in a failed state the controller could (in theory) behave differently.